packages/frontend/components/Footer/Footer.module.scss

```diff
@@ -1,7 +1,8 @@
-@import '../styles/variables';
+@import 'styles/variables';
 
 .footer {
   overflow: hidden;
+  margin-top: 20rem;
```

Comment:
Temporary? Seems rather large otherwise.

Reply:
Yeah temporary until we add the to the top button and to give the footer some whitespace